project/SparkBuild.scala

```diff
@@ -291,6 +291,12 @@ object SparkBuild extends PomBuild {
     publishLocal in SbtCompile := publishTask(publishLocalConfiguration in SbtCompile).value,
     publishLocal := Seq(publishLocal in MavenCompile, publishLocal in SbtCompile).dependOn.value,
 
+    javaOptions ++= {
```

Comment:
Apologies @luhenry there's a merge conflict around here. If you can resolve it I will merge.